src/Routes.js

```diff
@@ -25,25 +30,48 @@ export const routes = {
   executorDetails: '/:id/:run_id/:executor_id',
 };
 
-export const Routes = () => (
-  <Suspense fallback={<Fragment />}>
-    <Switch>
-      <Route exact path={routes.home} component={Home} />
-      <Route exact path={routes.details} component={RemediationDetails} />
-      <Route
-        exact
-        path={routes.runDetails}
-        render={(props) => <ActivityDetails remediation={{}} {...props} />}
-      />
-      <Route
-        exact
-        path={routes.executorDetails}
-        render={(props) => <ExecutorDetails {...props} />}
-      />
-      <Redirect path="*" to={routes.home} push />
-    </Switch>
-  </Suspense>
-);
+export const Routes = () => {
+  const [hasSystems, setHasSystems] = useState(true);
+  useEffect(() => {
+    try {
+      axios
+        .get(`${INVENTORY_TOTAL_FETCH_URL}?page=1&per_page=1`)
+        .then(({ data }) => {
+          setHasSystems(data.total > 0);
+        });
+    } catch (e) {
+      console.log(e);
+    }
+  }, [hasSystems]);
+
+  return !hasSystems ? (
+    <AsyncComponent
+      appName="dashboard"
+      module="./AppZeroState"
+      scope="dashboard"
+      ErrorComponent={<ErrorState />}
+      app="Remediation"
```

Comment:
I am not sure `Remediation` is registered with the zero state app names:

![image](https://user-images.githubusercontent.com/31385370/236172378-20a68c8a-1fb2-4197-95be-2f0276c6dfd5.png)

Running this PR results in error since the zero state components cannot find the associated with remediations content. Shall we add "remediations" first to insights-dashboard?

Reply:
I added a state, but we'll need to confirm the content as the mocks there are a bit confusing.